helm-values/beekeeper-load-test.yaml

```diff
@@ -0,0 +1,17 @@
+image:
+  pullPolicy: Always
+
+serviceAccount:
+  create: true
+  createRole: true
+
+config:
+  command: ['beekeeper', 'check', '--cluster-name=testnet', '--checks=load', '--create-cluster=false', '--metrics-pusher-address=pushgateway.staging.internal', '--timeout=12h', '--log-verbosity', '"5"']
```

Comment:
@vandot is there any way we can read the value for `--cluster-name` from the environment? maybe `'--cluster-name=$NAMESPACE'` or something